### PR TITLE
Update platform support

### PR DESCRIPTION
**Pull request by Eideren**
Some of those platforms do not work

See #308

@Jklawreszuk - do you think we should mark Linux as experimental as well ?

### CODE REVIEW

**Comment by Jklawreszuk** [comment]
@Eideren  Given that both the OpenGL and Vulkan graphics API's needs improvement I think it's worth adding Linux as well

**Comment by Eideren** [comment]
Done

File: index.html [hunk 1]
```diff
@@ -41,24 +41,6 @@ <h2 class="h3 mb-3 fw-bold text-stride">Accelerated development</h2>
                 </div>
             </div>
         </div>
-        <div class="col-lg-6">
-            <div class="card h-100">
-                <img src="/images/top/platforms.webp" alt="Robot" class="img-fluid p-3">
-                <div class="card-body pt-5 px-5">
-                    <h2 class="h3 mb-3 fw-bold text-stride">Develop once, deploy <br /> cross-platform</h2>
```
Comment (VaclavElias) [inline index.html:48]: Should we keep this section and make it like so?

```
                        {% include bullet-point.html text:'Windows Desktop' %}
                        {% include bullet-point.html text:'iOS and Android - experimental state' %}
                        {% include bullet-point.html text:'Linux - experimental state' %}
                        {% include bullet-point.html text:'macOS - experimental state' %}
```

Comment (Eideren) [reply index.html:48]: To me, we can't really advertise `Develop once, deploy cross-platform` when only one platform actually works. Nothing comes to mind right now, but if you can think of a way to keep that section while being honest about our capabilities, please do let me know and I'll update accordingly

Comment (VaclavElias) [reply index.html:48]: That is true. We should be crystal clear with such important information. All good then!

**Comment by VaclavElias** [comment]
Thank you @Eideren

**Pull request merged**